### DISCUSSION TITLE
Force rich to use the style I specity for braced output in f-strings

### DISCUSSION THREAD

**Discussion by njr0** [Textualize/rich#2072] [Q&A] [answered]
Probably a stupid newbie question, probably answered in the RTFM, but I couldn't see it quickly.

I'm doing this:

```python
console.print(f'{testsRun:5,} Run   ', style='black' end='')
console.print(f'{nf:5,} F   ', style='red' if nf else 'black', end='')
console.print(f'{ne:5,} E   ', style='purple' if ne else 'black', end='')
console.print(f'{ns:3} S    {c.__name__}')
```

and was hoping (and expecting) that the numbers in the f-strings would come out in the colour I specify. But they don't: they mostly come out in a sort of dull cyan (and bold, I think).

Can I force them to come out in the colour/style I say?

**Comment by willmcgugan** [comment, marked as answer]
Rich will highlight numbers and a bunch of other things automatically. You can switch that off with `highlight=False` on the `print` method, or in the Console constructor.

**Comment by njr0** [comment]
Perfect! And then I can leave the style as blank, rather than black, and get the default colour.

Brilliant. Sorry, I'm sure if I'd RTFM'd more I'd have found that.

Excellent stuff!

**Comment by njr0** [comment]
In case you're interested, this is what I end up with:

<img width="762" alt="Screenshot 2022-03-16 at 10 29 41" src="https://user-images.githubusercontent.com/117818/158570592-f7f631a6-08a5-44aa-a2c2-473958a7bb1c.png">

(It's a parallel test runner.)

I know it's not very colourful at the moment, but when I break the build, there'll be colour all over the place!
